internal/agent/device/console/console.go

```diff
@@ -204,31 +212,61 @@ func (c *ConsoleController) startForwarding(ctx context.Context, stdin io.WriteC
 }
 
 func (c *ConsoleController) bashProcess(ctx context.Context) (io.WriteCloser, io.ReadCloser, error) {
-	// TODO pty: this is how oci does a PTY:
-	// https://github.com/cri-o/cri-o/blob/main/internal/oci/oci_unix.go
-	//
-	// set PS1 environment variable to make bash print the default prompt
+
+	// create a new bash process
 	cmd := c.executor.CommandContext(ctx, "bash", "-i", "-l")
-	stdin, err := cmd.StdinPipe()
-	if err != nil {
-		return nil, nil, fmt.Errorf("error getting stdin pipe: %w", err)
-	}
-	stdout, err := cmd.StdoutPipe()
+	cmd.Env = append(cmd.Env, "TERM=xterm-256color")
+
+	// create a new PTTY
```

Comment:
should this be `PTY` everywhere ? Instead of `PTTY`

Reply:
oh, you are right, I thought it was "pseudo tty" but they just say pty everywhere.